data/mods/DinoMod/NPC/factions.json

```diff
@@ -0,0 +1,84 @@
+[
+  {
+    "type": "faction",
+    "id": "swamper",
+    "name": "Swampers Religious Community and Hotels and Casinos",
+    "likes_u": 30,
+    "respects_u": 20,
+    "known_by_u": false,
+    "size": 25,
+    "power": 20,
+    "currency": "icon",
+    "food_supply": 87500,
+    "wealth": 82500,
+    "relations": {
+      "free_merchants": {
+        "kill on sight": false,
+        "watch your back": true,
+        "share my stuff": true,
```

Comment:
Why this (and for the Tacoma Commune)?

Reply:
I see them as being fundamentally aligned with the Free Merchants, at least in terms of trade and survival. The Free Merchants themselves are doomed, but they're obvious recruits for the Swampers.